ironfish/src/rpc/server.ts

```diff
@@ -86,30 +94,51 @@ export class RpcServer {
   /** Authenticate the RPC request */
   authenticate(requestAuthToken: string | undefined | null): boolean {
     if (!requestAuthToken) {
-      this.logger.debug(`Missing Auth token in RPC request.`)
       return false
     }
 
-    const rpcAuthToken = this.internal.get('rpcAuthToken')
+    if (!this.authTokenBuffer.byteLength) {
+      return false
+    }
 
-    if (!rpcAuthToken) {
-      this.logger.debug(`Missing RPC Auth token in internal.json config.`)
+    if (requestAuthToken.length >= AUTH_MAX_LENGTH) {
```

Comment:
Nit: I think this should be a `>`, not a `>=`

Reply:
Fixed